src/components/Headers/AppHeader/AppHeader.tsx

```diff
@@ -19,7 +18,7 @@ const AppHeader = async () => {
     ? await getProfileByUserEmail(session.user.email)
     : null
 
-  if (session) {
+  if (session?.user?.id) {
```

Comment:
**nitpick:** We should leave as it is, and make session null if it's not present